### DISCUSSION TITLE
What is the best method to add an icon to a sidebar item

### DISCUSSION THREAD

**Discussion by shipswake** [facebook/docusaurus#10452] [General]
I have noticed a few methods being used, custom item, CSS, page title, to add an icon to a sidebar item.  Is there a best practice to add an icon?  Specifically type: doc

**Comment by shipswake** [comment]
Seems the easiest way it to use CSS.
custom.css:
.coming-soon > a::before {
  content: url(../img/Admin_20x20.svg);
  padding-right: 0.3rem;
}

sidebars.js:
    {
      type: 'doc',
      id: 'changelog',
      label: 'Changelog',
      className: 'coming-soon'
    },

**Reply by slorber** [reply]
Yes it's probably the easiest way

You can swizzle and plug in a React SVG component too but it's less ideal in particular if the icon is repeated multiple time the SVG markup will end up being duplicated in the final static page and it's better to use an external svg
